src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -37,6 +37,11 @@ import VariantCountCache from "./VariantCountCache";
 
 type PageMode = 'patient' | 'sample';
 
+class OncokbDefault implements IOncoKbData {

```

Comment:
I was actually just going to say I should put it in another file.  Maybe in model folder?